modules/solid_mechanics/include/materials/ReturnMappingModel.h

```diff
@@ -57,7 +57,6 @@ class ReturnMappingModel : public ConstitutiveModel
   Real _effective_strain_increment;
 
 private:
```

Comment:
And I would have said that even if you didn't bring in the cookies this morning.